data/models.py

```diff
@@ -8,6 +10,7 @@ class Contributor(models.Model):
     num_commits = models.IntegerField(default=None, null=True)
     reviews = models.IntegerField(default=None, null=True)
     issues_opened = models.IntegerField(default=None, null=True)
+    teams = JSONField(default=None, null=True)
```

Comment:
I see that you created another Model called Team in https://gitlab.com/coala/landing/blob/86203d6a9b5b840bc851804858744a4281240286/backend/org/models.py and use `teams = models.ManyToManyField(Team)` in Contributor model. Is there any reason for the use of JSONField here?

Reply:
>Is there any reason for the use of JSONField here?

Actually, I was using JSONField there; But, yesterday update that to ManyToManyField, and I still have to update it here.

Waiting for that to be merged anyway; Could you please review that one :)